bootstrap-osx.sh

```diff
@@ -28,6 +28,8 @@ cp /usr/local/lib/libav* libs/lib
 cp /usr/local/lib/libswscale* libs/lib
 cp /usr/local/lib/libqrencode* libs/lib
 cp /usr/local/lib/libsqlcipher* libs/lib
+cp /usr/local/lib/libfilteraudio* libs/lib
+cp /usr/local/lib/libtox* libs/lib
```

Comment:
It is.
Anyway, process described in INSTALL.md is broken for mac os, so I'll recall this changes.